src/style.css

```diff
@@ -1 +1,73 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  width: 100%;
+  position: sticky;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.logo {
+  position: absolute;
+  left: 50px;
+  top: 10px;
+}
+
+.nav {
+  height: 60px;
+  align-content: center;
+  margin-right: 50px;
```

Comment:
Don't set margins for BEM blocks, you can set padding: 0 50px; for header instead

Reply:
Done